lsdcGui.py

```diff
@@ -2035,6 +2040,7 @@ def createSampleTab(self):
         transmisionSPLabel = QtWidgets.QLabel("SetPoint:")
 
         self.transmission_ledit = self.transmissionSetPoint.getEntry()
+        self.transmission_ledit.setValidator(QtGui.QDoubleValidator(0,1))
```

Comment:
transmission of 0 doesn't really make sense. can you talk with beamline scientists to get reasonable minimums? this will require more digits for the validator.

Reply:
limits are now 0.001 and 0.999, 3 digit limit